data/pilots/first-order/tie-wi-whisper-modified-interceptor.json

```diff
@@ -0,0 +1,134 @@
+      "difficulty": "White", 
+      "linked": { "difficulty": "White", "type": "Rotate Arc" },
+      "type": "Focus" 
+    },
+    { "difficulty": "White", "type": "Evade" },
+    { "difficulty": "White", "type": "Lock" },
+    { 
+      "difficulty": "White",
+      "linked": { "difficulty": "White", "type": "Rotate Arc" },
+      "type": "Barrel Roll"
+    },
+    { 
+      "difficulty": "White",
+      "linked": { "difficulty": "White", "type": "Rotate Arc" },
+      "type": "Boost"
+    }
+  ],
+  "pilots": [
+    {
+      "name": "Kylo Ren",
+      "xws": "kyloren-tiewiwhispermodifiedinterceptor",
+      "initiative": 5,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Heavy Weapon Turret",
+        "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
+        "Talent",
+        "Missile",
+        "Tech",
+        "Configuration"
+      ],
+      "ability": "Before an enemy ship in your [Bullseye Arc] is dealt a facedown damage card, you may spend 1 [Force]. If you do, that damage card is dealt faceup instead.",
+      "ffg": -999,
+      "artwork": "???",
+      "image": "???",
+      "caption": "Supreme Leader of the First Order",
+      "keywords": ["TIE", "Dark Side"],
+      "force": { "value": 3, "recovers": 1, "side": ["dark"] }
+    },
+    {
+      "name": "Wrath",
+      "xws": "wrath",
+      "initiative": 5,
+      "limited": 1,
+      "shipAbility": {
+        "name": "Heavy Weapon Turret",
+        "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
+      },
+      "hyperspace": true,
+      "cost": 0,
+      "slots": [
+        "Talent",
+        "Talent",
+        "Missile",
+        "Tech",
+        "Configuration"
+      ],
+      "ability": "After you perform a [Bullseye Arc] attack, if you have 1 or more non-lock red or orange tokens, you may perform a bonus attack against a different target.",
+      "ffg": -999,
+      "artwork": "???",
+      "image": "???",
+      "caption": "Herald of Destruction",
+      "keywords": ["TIE"]
+    },
+    {
+      "name": "709th Legion Ace",
+      "xws": "709thlegionace",
+      "initiative": 4,
+      "limited": 0,
+      "shipAbility": {
+        "name": "Heavy Weapon Turret",
+        "text": "You can rotate your [Single Turret Arc] indicator only to your [Front Arc] or [Rear Arc]. You must treat the [Front Arc] requirement of your equipped [Missile] upgrades as [Single Turret Arc]."
+      },
+      "text": "Commanded by Supreme Leader Kylo Ren, the 709th Legion fights without mercy or remorse to crust the Resistance and bring the galaxy under the Forst Order's dominion.",
```

Comment:
Text should be: "Commanded by Supreme Leader Kylo Ren, the 709th Legion fights without mercy or remorse to crush the Resistance and bring the galaxy under the First Order's dominion."